src/browser-polyfill.js

```diff
@@ -491,19 +491,16 @@ if (typeof browser === "undefined" || Object.getPrototypeOf(browser) !== Object.
       get: {minArgs: 1, maxArgs: 1},
       set: {minArgs: 1, maxArgs: 1},
     };
-    apiMetadata.privacy = {
-      network: {
-        networkPredictionEnabled: settingMetadata,
-        webRTCIPHandlingPolicy: settingMetadata,
-      },
-      services: {
-        passwordSavingEnabled: settingMetadata,
-      },
-      websites: {
-        hyperlinkAuditingEnabled: settingMetadata,
-        referrersEnabled: settingMetadata,
-      },
-    };
+    // Wrap all "network", "services", "websites" APIs in chrome.privacy.*
+    if (extensionAPIs.privacy) {
+      apiMetadata.privacy = {};
+      for (const privacyType of ["network", "services", "websites"]) {
+        apiMetadata.privacy[privacyType] = {};
+        for (const privacyName in extensionAPIs.privacy[privacyType]) {
```

Comment:
This would also include enumerable properties from `Object.prototype`, which shouldn't be exported.

Reply:
Good point, will update.
Guess using `Object.keys()` might be better.

Reply:
Done.